pkg/network/ebpf/c/protocols/classification/shared-tracer-maps.h

```diff
@@ -7,7 +7,7 @@
 
 // Maps a connection tuple to its classified protocol. Used to reduce redundant
 // classification procedures on the same connection
-BPF_HASH_MAP(connection_protocol, conn_tuple_t, protocol_stack_t, 0)
+BPF_LRU_MAP(connection_protocol, conn_tuple_t, protocol_stack_t, 0)
```

Comment:
suggestion: Add here a line of documentation explaining about the "user mode" modifications in case of LRU not supported

Reply:
Good point 👍